DockerFile-test

```diff
@@ -0,0 +1,34 @@
+FROM ipfs/go-ipfs:release
```

Comment:
Name should be `Dockerfile-test` - not `DockerFile-test`. Just matter of consistency.